admin-portal/src/pages/supplierQuotationUpload/supplierQuotationUpload.css

```diff
@@ -0,0 +1,67 @@
+:root {
+  --elixir-leading-desktop: 1.25;
+  --elixir-leading: calc(var(--elixir-leading-desktop) * 1rem);
+}
+
+.pageContent {
```

Comment:
Use a parent class instead of directly querying selectors. So that we can avoid overriding styles for elements that we didn't intend to. 

ex:

`.supplierQuotationUpload .pageContent`

Reply:
PS: Since we are not using css modules, whatever styles you write here affects globally.